Sources/o/File/o+file.swift

```diff
@@ -99,6 +99,27 @@ public extension o.file {
 
         return try JSONDecoder().decode(Value.self, from: data)
     }
+
+    /// Read a file's data as the type `String`
+    ///
+    /// - Parameters:
+    ///   - path: The path to the directory containing the file. The default is `.`, which means the current working directory.
+    ///   - filename: The name of the file to read.
+    ///   - encoding: The String.Encoding to use to get the string.
+    /// - Returns: The file's data decoded as an instance of `String`.
+    /// - Throws: If there's an error reading the file or decoding its data.
+    static func string(
+        path: String = ".",
+        filename: String,
+        encoding: String.Encoding = .utf8
+    ) throws -> String? {
```

Comment:
So it can throw and return an optional, which means the call needs to unwrap an optional and accommodate a possible exception. 

What do you think about either 1. throwing when the data can't be converted to a string, or 2. using `try?` to convert the exception to an optional? I

```suggestion
    ) throws -> String {
```

```suggestion
guard let string = String(data: data, encoding: encoding) else {
  struct CouldNotConvertDataToString: Error {}
  throw CouldNotConvertDataToString()
}

return string
```

Reply:
Great suggestion! I will remove the optional String return in favor of throwing an error.

Reply:
Updated in https://github.com/0xOpenBytes/o/pull/7/commits/55dd8504a818b1be71f5c0edcf817ef2e77e6e77